src/client/components/SidebarMenu/SidebarMenu.stories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import SidebarMenu from './SidebarMenu';
+
+export default {
+  title: 'SidebarMenu',
+  component: 'SidebarMenu',
```

Comment:
'SidebarMenu' in component should be without single quotes.

Reply:
thanks , it is changed.